src/averageAges.js

```diff
@@ -15,11 +15,25 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  for (const person of people) {
+    person.age = person.died - person.born;
+    person.century = Math.ceil(person.died / 100);
+  }
+
+  const countOfMen = people.reduce((count, person) => {
```

Comment:
Replace the if statement with ternary operator

```suggestion
  const countOfMen = people.reduce((count, person) => 
  (person.sex === 'm' && person.century === century) || 
  (person.sex === 'm' && !century) ? count + 1 : count
, 0);
```